app/workers/parse_raw_event_job.rb

```diff
@@ -28,16 +28,32 @@ def perform(raw_event_id)
         raw_event_id: re.id
       )
 
+      producer = Poseidon::Producer.new(
+        ["#{LensServer.config.kafka.host}:#{LensServer.config.kafka.port}"],
+        "lens_bg_producer"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.